Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberViewModel.swift

```diff
@@ -35,13 +35,15 @@ extension CardNumberViewModel: CardNumberViewModelProtocol {
 
     switch cardValidator.validateCompleteness(cardNumber: cardNumber) {
     case .failure:
+        delegate?.update(cardNumber: nil, scheme: nil)
         return nil
     // An unknown scheme can also be generated when an eager validation was matched
     //    but the number is not complete.
     // In this case we only update delegate if we have a final result to avoiding
     //    overriding the eager validation with a less informative update
     case .success((let isComplete, let scheme)):
       guard isComplete else {
+          delegate?.update(cardNumber: nil, scheme: nil)
```

Comment:
There are a lot of nils passed around. Why can't we just use the `cardNumber` as its been determined and different scheme, like `unknown`? Seems excessive to pass nil so frequently.